core/table.ts

```diff
@@ -82,6 +84,11 @@ export class Table {
     if (config.bigquery) {
       this.bigquery(config.bigquery);
     }
+    if (config.tags)
+    {
```

Comment:
nit: not sure what editor you are using, but fwiw we tend to use VSCode with the Prettier plugin, it should autoformat things like this on save. anyways, move this `{` to the previous line?

Reply:
I use VSCode. Thanks for the tip.